packages/shared/providers.yaml

```diff
@@ -5169,7 +5169,7 @@ salesforce:
             description: The instance URL of your Salesforce account
             format: uri
             pattern: '^https?://.*$'
-            optional: true
+            automated: true
 
```

Comment:
Should update https://github.com/NangoHQ/nango/blob/master/packages/shared/providers.yaml#L2643 and github-app-oauth as well

Reply:
both appLink and installationId or just installationId?

Reply:
Just installationId

Reply:
ah yes 🤦🏻